### PR TITLE
Minor change to c#

### PR DESCRIPTION
**Pull request by pitermarx**

### CODE REVIEW

File: csharp/optimized/Program.cs [hunk 1]
```diff
@@ -17,22 +17,26 @@ public sealed class Ref<T> {
...
-        while ((line = Console.ReadLine()) != null)
+        using (var stream = Console.OpenStandardInput())
+        using (var reader = new StreamReader(stream))
         {
-            line = line.ToLowerInvariant();
-            var words = line.Split(' ', StringSplitOptions.RemoveEmptyEntries);
-            string word;
-            for (int i = 0; i < words.Length; i++)
+            string line = null;
+            while ( (line = reader.ReadLine()) != null)
```
Comment (benhoyt) [inline csharp/optimized/Program.cs:24]: Nit: extra space between `( (`.

**Comment by benhoyt** [comment]
Thanks for the contribution. What do these changes actually do? (I'm not a C# guy, so wondering whether they're just cosmetic or they speed things up somehow.)

**Comment by benhoyt** [comment]
I think this is moot now that I've merged https://github.com/benhoyt/countwords/pull/59 - closing

**Comment by pitermarx** [comment]
It's basically the same, but does not instantiate a SyncReader
https://referencesource.microsoft.com/#mscorlib/system/console.cs,b43a20e3d81a7216
thks

**Pull request closed without merging**